### DISCUSSION TITLE
Performance on android ?

### DISCUSSION THREAD

**Discussion by blaize** [fyne-io/fyne#4057] [Q&A] [answered]
I use a widget.List and populate with about 100 rows. Each row contain a checkbox, 2 labels and 1 icon.
On windows, this work well (core i7 12th gen, sure it will work!). On android smartphone (pixel 7a), it's very very laggy scrolling (almost unusable)
I can't pretend to distribute an app with this performances. Does this bug will be fixed ? Or it's a limitation of the toolkit and I have to find another solution ?

Thanks, it will be so bad if I have to give up with fyne due to the performances. Really love to work with this toolkit.

https://github.com/fyne-io/fyne/assets/1087443/fd56e65d-910e-491a-8793-64793fe76fab

**Comment by andydotxyz** [comment, marked as answer]
It is a bug specific to Android which we want to fix, any help with that would be great - it's not clear which graphics operation is lagging on those devices.

**Reply by Wuvist** [reply]
has this been fixed? or is there a issue linked to this bug?

**Reply by Wuvist** [reply]
BTW, I've just tested long list scrolling on Android, the experience is slightly better than above video, but "laggy" compared to normal mobile scroll with acceleration.

**Reply by andydotxyz** [reply]
You probably want this issue: https://github.com/fyne-io/fyne/issues/4058
